akka-docs/src/main/paradox/persistence-query-leveldb.md

```diff
@@ -5,6 +5,14 @@ As a replacement we recommend using [Akka Persistence JDBC](https://doc.akka.io/
 
 ## Dependency
 
+The Akka dependencies are available from Akka's library repository. To access them there, you need to configure the URL for this repository.
+
+@@repository [sbt,Gradle,Maven] {
```

Comment:
Should it be [sbt, Maven, Gradle] like most of them? 

There are a few more in the following files with the same difference.